components/apimgt/org.wso2.carbon.apimgt.rest.api.util/src/main/java/org/wso2/carbon/apimgt/rest/api/util/utils/RestApiUtil.java

```diff
@@ -1400,4 +1400,19 @@ public static boolean isDevPortalAnonymousEnabled(String tenantDomain) {
     public static String getOrganization(MessageContext ctx) {
         return (String) ctx.get(RestApiConstants.ORGANIZATION);
     }
+
+    /**
+     * Method to extract the organization with validation
+     * @param ctx MessageContext
+     * @return organization
+     */
+
+    public static String getOrganizationWithValidation(MessageContext ctx) throws APIManagementException{
```

Comment:
What if we name the method `getValidatedOrganization()` ?

Reply:
Fixed via https://github.com/wso2/carbon-apimgt/pull/10726/commits/e665e6479c5d7b3b3ade8f735eb21615e02557f4